x-pack/platform/plugins/shared/lens/public/persistence/basic_lens_client.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { HttpStart } from '@kbn/core/public';
+import type { ContentManagementPublicStart } from '@kbn/content-management-plugin/public';
+import type {
+  SerializableAttributes,
+  BasicVisualizationClient,
+} from '@kbn/visualizations-plugin/public';
+
+import type { LensAttributes } from '../../server/content_management';
+import { LensClient } from './lens_client';
+import { getLensSOFromResponse } from './utils';
+
+/**
+ * This is a wrapper client used only to update basic attributes from the vis plugin
+ */
+export function getLensBasicClient<Attr extends SerializableAttributes = SerializableAttributes>(
+  cm: ContentManagementPublicStart,
+  http: HttpStart
+): BasicVisualizationClient<'lens', Attr> {
+  const lensClient = new LensClient(http);
```

Comment:
This is a new Basic Lens client to be used to load and update basic attributes from the **Visualize Library** page. Since we updated the Lens client to use the new APIs we need to monkey patch this basic client to save the Lens SO using the new Lens Client.